docs/src/getting-started/basics.md

```diff
@@ -43,8 +43,8 @@ abigen!(
     "your_project/out/debug/contract_test-abi.json",
 );
 
-// This helper will will launch a local node and provide a test wallet linked to it
-let wallet = launch_provider_and_get_wallet().await;
+// This helper will will launch a local node and provide 10 test wallets linked to it
+let wallet = launch_provider_and_get_wallets(WalletConfig::default()).await;
```

Comment:
`wallet` is a `Vec<LocalWallet>` so it's not going to work with the `deploy` call below.